### ISSUE TITLE
ISSUES IN RUNNING SOMATIC PREDICTIONS

### ISSUE THREAD

**Issue by Pritam123456** [ikalatskaya/ISOWN#7]
after completing all the dependencies and all the steps of dbSNP etc we have reached  in step 1. But in step 1 we faced a big problem. we ran the command: 
for i in `ls ${ISOWN_HOME}/test_data/*vcf`; do perl ${ISOWN_HOME}/bin/database_annotation.pl $i `basename $i` ; done

but it shows us this lines in the command promt: 

annotating input file with ANNOVAR ...Can't open perl script "/home/sakib/ISOWN/bin/../external_tools/annovar_2012-03-08/annotate_variation.pl": No such file or directory
Couldn't open 3d2edf87-6ec5-4c9f-9212-e8a751cc33e8.dkfz-snvCalling_1-0-132-1.20160126.vcf.temp.annovar.vcf.temp.convert2annovar.variant_function/

The dbSNP 142 file is not found.  Please correct the path in /home/sakib/ISOWN/bin/database_annotation.pl and try again - see path below:

	/home/sakib/ISOWN/bin/../external_databases/dbSNP142_All_20141124.vcf.gz.modified.vcf.gz

annotating input file with ANNOVAR ...Can't open perl script "/home/sakib/ISOWN/bin/../external_tools/annovar_2012-03-08/annotate_variation.pl": No such file or directory
Couldn't open 519b8381-95d5-4fce-a90c-7576cce2110c.dkfz-snvCalling_1-0-132-1.20160126.vcf.temp.annovar.vcf.temp.convert2annovar.variant_function/

The dbSNP 142 file is not found.  Please correct the path in /home/sakib/ISOWN/bin/database_annotation.pl and try again - see path below:

	/home/sakib/ISOWN/bin/../external_databases/dbSNP142_All_20141124.vcf.gz.modified.vcf.gz

Now, what can we do sir? please help

**Comment by batis2ta** [comment]
Hi, please know that I am a user. to fix this you can change the name of you vcf file and change the directory of annovar .pl files to the mentioned directory: external_tools/annovar_2012-03-08/

**Comment by qtrinh** [comment]
Yes, it seems annovar_2012-03-08 is missing.  Please follow instructions from here https://github.com/ikalatskaya/ISOWN/blob/master/DEPENDENCIES.md
